common/app/views/fragments/commercial/containerLogo.scala.html

```diff
@@ -0,0 +1,14 @@
+@(branding: common.commercial.Branding, isOnTheLeft: Boolean)(implicit request: RequestHeader)
+@import common.commercial.PaidContent
+
+<div class="badge @if(isOnTheLeft){badge--alt}">
+    @branding.label
+    <a class="badge__link" href="@branding.sponsorLink" data-link-name="logo link">
+        <img src="@branding.sponsorLogo.url"
```

Comment:
I thought amp was just for content pages.  But I'll add the amp variant.  So I suppose it should go back on the card logo as well.